txp/Pos/Txp/Core/Types.hs

```diff
@@ -338,7 +347,7 @@ mkTxPayload txws = do
 ----------------------------------------------------------------------------
 
 -- | Particular undo needed for transactions
```

Comment:
It would be good to write what exactly this `TxUndo` means (had to be written long ago) and why it can contain `Nothing`s.